flight/Modules/OnScreenDisplay/onscreendisplay.c

```diff
@@ -1618,12 +1621,13 @@ int32_t OnScreenDisplayInitialize(void)
 	has_gps = PIOS_Modules_IsEnabled(PIOS_MODULE_GPS);
 	has_battery = module_settings.AdminState[MODULESETTINGS_ADMINSTATE_BATTERY];
 
-	uint8_t filter;
-	StateEstimationNavigationFilterGet(&filter);
-	if (filter != STATEESTIMATION_NAVIGATIONFILTER_NONE) {
-		has_nav = true;
-	} else {
-		has_nav = false;
+	has_nav = false;
+	if (StateEstimationHandle()) {
+		uint8_t filter;
+		StateEstimationNavigationFilterGet(&filter);
+		if (filter != STATEESTIMATION_NAVIGATIONFILTER_NONE) {
```

Comment:
Yes, but there is also no StateEstimationHandle so it doesn't get that far (although that's a trap for it one day when it might).